templates/vue/src/components/Toolbar/TapestryOperationsButton.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <b-dropdown
+    id="operations-button"
+    right
+    variant="none"
+    toggle-class="operations-button"
+    :toggle-attrs="{
+      'aria-label': 'Tapestry Operations',
+    }"
+    menu-class="operations-menu"
+  >
+    <template #button-content>
+      <i class="fas fa-ellipsis-h"></i>
+    </template>
+    <b-dropdown-item-button @click="open(names.EXPORTDUPLICATE)">
+      Export/Duplicate Tapestry
+    </b-dropdown-item-button>
+    <b-dropdown-item-button @click="open(names.OTHEROPERATIONS)">
+      Other Operations
+    </b-dropdown-item-button>
+    <export-duplicate-modal
+      :show="openOperation === names.EXPORTDUPLICATE"
+      @close="close(names.EXPORTDUPLICATE)"
+    ></export-duplicate-modal>
+    <other-operations-modal
+      :show="openOperation === names.OTHEROPERATIONS"
+      @close="close(names.OTHEROPERATIONS)"
+    ></other-operations-modal>
```

Comment:
Best if these are nested outside of `b-dropdown`

Reply:
Sounds good, that makes sense. 7dba59b should fix this.